### PR TITLE
week 4 fan page

### PR DESCRIPTION
**Pull request by pyasger**
week 4 fan page

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    padding: 10px;
+}
+
+figcaption {
+    text-align: center;
+    font-family: papyrus, fantasy;
+}
+
+h1 {
+    background-color: #mediumpurple;
```
Comment (mbMosman) [inline style.css]: This is not valid.  Make sure to validate the CSS as well.

image

File: index.html [hunk 2]
```diff
@@ -1 +1,31 @@
...
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>best water</title>
+<meta charset="utf-8">
+<link rel="stylesheet" href="style.css">
+</head>
+    <h1>Water</h1>
+    <nav>
+        <b>
```
Comment (mbMosman) [inline index.html]: You should be doing this through CSS, so the `<b>` should be removed.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    min-width: 700px;
+    max-width: 2048px;
+    padding-left: 20px;
+}
+
+figure {
+    margin-top: 30px;
+    width: 480px;
+    border: 1px solid #000000;
+    padding: 10px;
```
Comment (mbMosman) [inline style.css]: A background color of white would have completed the effect.

**Comment by pyasger** [comment]
completed

File: index-alt.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+<head>
+    <title>best water</title>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <h1>Water</h1>
+    </header>
+    <nav>
```
Comment (mbMosman) [inline index-alt.html]: I see a nav in the HTML but not on the page....

**Pull request closed without merging**